src/pages/SearchPage/index.js

```diff
@@ -0,0 +1,181 @@
+import PropTypes from 'prop-types';
+import React, {useEffect, useMemo, useState} from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import {usePersonalDetails} from '@components/OnyxProvider';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import useDebouncedState from '@hooks/useDebouncedState';
+import useLocalize from '@hooks/useLocalize';
+import useNetwork from '@hooks/useNetwork';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import * as OptionsListUtils from '@libs/OptionsListUtils';
+import Performance from '@libs/Performance';
+import * as ReportUtils from '@libs/ReportUtils';
+import reportPropTypes from '@pages/reportPropTypes';
+import * as Report from '@userActions/Report';
+import Timing from '@userActions/Timing';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import SearchPageFooter from './SearchPageFooter';
+
+const propTypes = {
+    /* Onyx Props */
+
+    /** Beta features list */
+    betas: PropTypes.arrayOf(PropTypes.string),
+
+    /** All reports shared with the user */
+    reports: PropTypes.objectOf(reportPropTypes),
+};
+
+const defaultProps = {
+    betas: [],
+    reports: {},
+};
+
+const searchRendered = () => {
```

Comment:
Maybe having eg. `setPerformanceTimersEnd()` (I'm terrible at this sorry 😂) would be more precise, what do you think? The naming guidelines can come in handy for naming these.

Reply:
I concur, I just used previous name :)